packages/flutter_tools/lib/src/commands/format.dart

```diff
@@ -28,6 +28,10 @@ class FormatCommand extends FlutterCommand {
       defaultsTo: false,
       negatable: false,
     );
+    argParser.addOption('line-length',
+      abbr: 'l',
+      help: 'Wrap lines longer than this.',
```

Comment:
I'd word this differently (there's also a global --wrap-column for wrapping command output).

Something like "Reformats code to produce lines shorter than this length."

Reply:
I have modified command words.  I am so sorry for my english.

Reply:
Please do not worry about your English. I'm sure my Chinese is FAR worse. :-)

Sorry, I wasn't clear: I just meant that the description shouldn't have the word "wrap" in it, to avoid confusion. I wasn't asking you to change the name of the option.

Please change the name of the option back to `--line-length`.  We don't want two options with the same name and different meanings.

Reply:
I've fixed it. Thank you!